quipucords/api/details_report/model.py

```diff
@@ -15,7 +16,7 @@ class DetailsReport(models.Model):
     )
     report_version = models.CharField(max_length=64, null=False)
     report_platform_id = models.UUIDField(default=uuid.uuid4, editable=False)
-    sources = models.JSONField(null=False, default=list)
+    sources = models.JSONField(null=False, default=list, encoder=RawFactEncoder)
```

Comment:
Why is this change part of this PR? How exactly does the sources model encoder affect collection of Ansible logs?

Are this and its related changes (like the test) what you suggested would be skipped when we port this change to 1.3?

Reply:
this is here on the same commit that introduces the automatic generation of raw facts to details report. as I pointed in the commit message
>refactor(DetailsReportFactory): generate faker-powered raw facts on the fly
>
>For convenience, DetailsReport.source is now using the same RawFact
    encoder used for raw facts. Even though this is not required for production
    code, it is technically the correct thing to do, since that field
    holds raw facts as well.
    
    
And yes, this is part of what I'd skip when backporting to 1.3

Reply:
I was reviewing the whole PR, not individual commits, and didn't see this message. Thank you for pointing that out.

Reply:
To be more concise: without this addition the factory for details report would break sometimes (when it randomly decide to pick OCP facts)